learning-to-learn/learning-to-learn.md

```diff
@@ -87,4 +87,7 @@ Take the [Growth Mindset Self-Assessment](http://mindsetonline.com/testyourminds
 
 ### Check for Understanding
 
-Participants write a self-reflective journal entry covering: 1) What they learned about themselves, 2) What from this lesson they felt challeneged by, 3) What from this lesson they want to practice.
+Participants write a self-reflective journal entry covering: 
+1. What they learned about themselves
+1. What from this lesson they felt challeneged by
```

Comment:
Looks good! A small typo in challenged

Reply:
We are not actually editing this file; we are checking the slideshows, @manaswinidas

Reply:
@manaswinidas @hoduchieu01 @Kundan28 please be sure to read the PR descriptions